2_run/src/run_glm3.R

```diff
@@ -86,21 +85,24 @@ run_glm3_model <- function(sim_dir, nml_objs, model_config, burn_in, burn_out, e
   lake_id <- model_config$site_id
   time_period <- model_config$time_period
   gcm <- model_config$gcm
+  meteo_fl <- model_config$meteo_fl
   
   # prepare to write inputs and results locally for quick I/O
   sim_lake_dir <- file.path(sim_dir, sprintf('%s_%s_%s', lake_id, gcm, time_period))
   dir.create(sim_lake_dir, recursive=TRUE, showWarnings=FALSE)
   # delete sim_lake_dir after model has run and we've extracted the results
   on.exit(unlink(sim_lake_dir, recursive = TRUE))
+  
+  # Read in meteo_data
+  meteo_data <- arrow::read_feather(meteo_fl) %>%
+    mutate(time = as.Date(time))
```

Comment:
So we are reading this file in twice? Once here and once in `combine_glm_output()`?

Reply:
Nevermind I see. Commented there instead. Consider the need for `as.Date()` here, too. Mine is already a `Date` class when I use `arrow::read_feather()`